demos/smart_classroom_demo_gapi/include/action_detector.hpp

```diff
@@ -108,25 +85,21 @@ struct ActionDetectorConfig : public CnnConfig {
 };
 
 
-class ActionDetection : public AsyncDetection<DetectedAction>, public BaseCnnDetection {
+class ActionDetection  {
 public:
     explicit ActionDetection(const ActionDetectorConfig& config);
 
-    void submitRequest() override;
-    void enqueue(const cv::Mat &frame) override;
-    void wait() override { BaseCnnDetection::wait(); }
-    void printPerformanceCounts(const std::string &fullDeviceName) override {
-        BaseCnnDetection::printPerformanceCounts(fullDeviceName);
-    }
-    DetectedActions fetchResults() override;
+    DetectedActions fetchResults(const cv::Mat &in_ssd_local,
+                                 const cv::Mat &in_ssd_conf,
+                                 const cv::Mat &in_ssd_priorbox,
+                                 const cv::Mat &in_ssd_anchor1,
+                                 const cv::Mat &in_ssd_anchor2,
+                                 const cv::Mat &in_ssd_anchor3,
+                                 const cv::Mat &in_ssd_anchor4,
+                                 const cv::Mat &in_frame);
```

Comment:
I'd replace these eight arguments with a single structure.

Reply:
I put it in `vector<cv::Mat>` ssd_result.